### PR TITLE
Black pepper tofu

### PR DESCRIPTION
**Pull request by matthewdking**
relates #40

### CODE REVIEW

File: recipes/main/black-pepper-tofu.md
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md:2]: This badge is broken

**Update** After a refresh it appears to be working. It's not a repeatable bug but the url looks a little weird so there might be something wrong :woman_shrugging:

File: recipes/main/black-pepper-tofu.md
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: You showed me a delicious looking photos of the recipe that would look great in the readme :+1:

File: recipes/main/black-pepper-tofu.md
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: A nod to the original author ([Yotam Ottolenghi](https://www.ottolenghi.co.uk/black-pepper-tofu-a-shop)) is always appreciated. I for one think he'd be flattered to be included in such a prestigious cooking forum.

File: recipes/main/black-pepper-tofu.md
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: Packs in the UK vary in size quite a lot. Definitely not a block to merging though :+1:

**Review by finnhodgkin — CHANGES_REQUESTED** [review]

File: recipes/main/black-pepper-tofu.md
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: Was it actually one bulb of garlic?! Like I believe you I'm just wanted to express my surprise

File: recipes/main/black-pepper-tofu.md
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md:41]: nice tip 👍

File: recipes/main/black-pepper-tofu.md
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: I would say normally with soy sauce you don't need a measurement but with this one I remember you used a lot so maybe some guidance here would be helpful

File: recipes/main/black-pepper-tofu.md
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: Could change this to Rice/Noodles/Side Dish?

Although don't know if you need this line at all, the line in the intro does enough? Most recipes don't have a part saying make sure the other stuff is ready at the same time

File: recipes/main/black-pepper-tofu.md
Comment (mattlub) [reply recipes/main/black-pepper-tofu.md]: Does he have GitHub though???

**Review by m4v15 — CHANGES_REQUESTED** [review]
maybe don't need to change, just had see my questions

File: recipes/main/black-pepper-tofu.md
Comment (matthewdking) [reply recipes/main/black-pepper-tofu.md]: I think it was more like half! Maybe I will add a specific number of cloves

**Comment by bradreeder** [comment]
I'm making this now (thanks, and hi from London 😋 ) and I don't think there's anything in the recipe telling you what to do with the chillies? (Assuming to add to the sauce along with the onion, garlic, and ginger.)

**Comment by matthewdking** [comment]
@bradreeder good spot! Remove the seeds from the chillies and then finely chop them and add them with the onion garlic and ginger. I will add it to the recipe now! Hope you enjoy it :man_cook: :fork_and_knife:

**Comment by bradreeder** [comment]
@matthewdking I think I went overboard on the chillies 🔥 🚒 but v nice thanks, have enough leftover for the weekend.

**Comment by yvonne-liu** [comment]
@bradreeder you should star this repo ⭐️ !

File: recipes/main/black-pepper-tofu.md
Comment (Karyum) [inline recipes/main/black-pepper-tofu.md]: I'm not sure you could say it was a hit with everyone, thats just a broad statement.

File: recipes/main/black-pepper-tofu.md
Comment (Karyum) [inline recipes/main/black-pepper-tofu.md]: on line 39 you have corn flour with a pace between them.
consistency

**Review by Karyum — CHANGES_REQUESTED** [review]
Also add the links to the README

File: recipes/main/black-pepper-tofu.md
Comment (yvonne-liu) [reply recipes/main/black-pepper-tofu.md]: What is a pace?

Comment (m4v15) [reply recipes/main/black-pepper-tofu.md]: he's right tbf, should be cornflour everywhere

Comment (Karyum) [reply recipes/main/black-pepper-tofu.md]: @yvonne-liu I meant space but 
http://www.dictionary.com/browse/pace

File: recipes/main/black-pepper-tofu.md
Comment (matthewdking) [reply recipes/main/black-pepper-tofu.md]: Who was it not a hit with?

Comment (Karyum) [reply recipes/main/black-pepper-tofu.md]: well me atleast

**Review by Karyum — APPROVED** [review]

**Review by finnhodgkin — APPROVED** [review]

File: README.md
Comment (yvonne-liu) [inline README.md]: The inconsistency with capitalizing with the rest of the names is 🙅 .

**Review by yvonne-liu — CHANGES_REQUESTED** [review]

File: README.md
Comment (yvonne-liu) [reply README.md]: Might as well change it to "Black Beans with Lime and Coriander Rice" as well.

**Review by yvonne-liu — APPROVED** [review]

**Comment by m4v15** [comment]
@yvonne-liu errr you should wait until everyone has approved the pull request before merging m8

**Comment by yvonne-liu** [comment]
@m4v15 "Once a pull request has been reviewed by 3 people who ate the meal, as well as or including a maintainer, it can be merged." 👋

**Comment by m4v15** [comment]
@yvonne-liu Firstly, that comes under the section titles "Instructions for maintainers" which you are not. Second of all, even if it didn't, the "3 people who ate the meal" criteria wasn't fulfilled here as @finnhodgkin did not, and *third* of all, I would *think* it is safe to assume/implied knowledge that you don't merge a PR when someone has requested changes and is yet to approve it.

**Comment by m4v15** [comment]
I mean, I was joking at first, but now this is serious

**Comment by m4v15** [comment]
Oh also I mean just cos I had requested changes and not reviewed it, not all the other people who hadn't reviewed it

**Comment by m4v15** [comment]
you can apologise here or on WhatsApp, either works ❤️

**Comment by yvonne-liu** [comment]
I'd _consider_ apologizing, but I'll never apologise 💔

**Comment by m4v15** [comment]
that'll do

**Comment by mattlub** [comment]
> you don't merge a PR when someone has requested changes and is yet to approve it

@yvonne-liu agree with @m4v15 about this, should it be stated in the guidelines?

**Pull request merged**
